packages/lesswrong/lib/collections/tags/helpers.ts

```diff
@@ -21,19 +21,20 @@ export const tagMinimumKarmaPermissions = forumSelect({
 type GetUrlOptions = {
   edit?: boolean,
   flagId?: string
+  tab?: string
 }
 
 export const tagUrlBase = taggingNameIsSet.get() ? taggingNamePluralSetting.get() : 'tag'
 export const tagCreateUrl = `/${tagUrlBase}/create`
 export const tagGradingSchemeUrl = `/${tagUrlBase}/tag-grading-scheme`
 
-export const tagGetUrl = (tag: {slug: string, isSubforum: boolean}, urlOptions?: GetUrlOptions) => {
-  // Assume links that are not explicitly for the subforum should go to the wiki tab (this may change in the future)
-  const urlSearchParams = tag.isSubforum ? {tab: "wiki", ...urlOptions} : urlOptions
```

Comment:
This function already appended `?tab=wiki` to the url, so just removing this and making the plain url default to the wiki should keep all the links the same